cmake/toolchains/c.windows.x86_64.cmake

```diff
@@ -1,13 +1,11 @@
-
 # this one is important
 SET(CMAKE_SYSTEM_NAME Windows)
 SET(CMAKE_SYSTEM_PROCESSOR x86_64)
 
 # specify the cross compiler
-SET_CCACHE(CMAKE_C_COMPILER x86_64-pc-mingw32-gcc)
-SET_CCACHE(CMAKE_CXX_COMPILER x86_64-pc-mingw32-g++)
-SET_CCACHE(CMAKE_RC_COMPILER x86_64-pc-mingw32-windres)
-
```

Comment:
Ok so I added the CCache stuff in CmakeLists.txt now. The macros cannot be used at this point!